twitter/views.py

```diff
@@ -122,8 +122,8 @@ def _filter_results(request, campaign):
         filter_metrics = ['%d' % int(i) for i in request.POST.getlist('filter_metrics')]
     if 'filter_entities' in request.POST:
         filter_entities = request.POST.getlist('filter_entities')
-    if 'fiter_data_center' in request.POST:
-        filter_data_center = request.POST.getlist('fiter_data_center')
+    if 'filter_data_center' in request.POST:
+        filter_data_center = request.POST.getlist('filter_data_center')
```

Comment:
This is an api change?

Reply:
im sorry i don't understand... can you be more clear please?

Reply:
Sometimes people write code with third party consumers, if so, a change like that could break such consumers.

I try to call out such changes in case the app is worried about such things.

If you don't have external consumers, then there's nothing to worry about.

I make PRs like this regularly, often w/ hundreds of changes. Changes that impact code outside the repository tend to require a closer look / more consideration.

Reply:
Ok I see now, thanks! Actually it's a feature currently not used, so all is fine ;) thanks for the PR